package.json

```diff
@@ -624,9 +630,10 @@
 		"@asciidoctor/docbook-converter": "2.0.0",
 		"@orcid/bibtex-parse-js": "0.0.25",
 		"asciidoctor-kroki": "0.15.4",
+		"pug": "^3.0.2",
```

Comment:
We should not add `pug` as a dependency in the Asciidoctor VS Code extension. Template engines are optional and dependencies should be installed by users.

We should probably mention that in the setting description and in the README.